spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/serializer/DataConverter.scala

```diff
@@ -431,17 +520,21 @@ object DataConverter extends DataConverter{
           attributeBuilder.setTensorValue(storages.get(tensorId).get
             .asInstanceOf[BigDLTensor])
         } else {
-          val tensorNumeric = tensor.getTensorNumeric()
-          val offSet = tensor.storageOffset()
           val totalElement = tensor.nElement()
           val dimension = tensor.dim()
-          val isScalar = tensor.isScalar
           val tensorBuilder = BigDLTensor.newBuilder
           tensorBuilder.setId(tensorId)
           tensorBuilder.setDimension(dimension)
-          tensorBuilder.setOffset(offSet)
```

Comment:
why remove this ?

Reply:
For `QuantizedTensor`, there's no such methods supporting. So there're in next lines for `DenseTensor`.